packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -1219,6 +1234,9 @@ class DateRangePickerDialog extends StatefulWidget {
   /// If `null`, the date of `DateTime.now()` will be used.
   final DateTime? currentDate;
 
+  /// The optional [DateFormat] to be used for the text input.
```

Comment:
Agreed, I added this here and also I realized I had made a mistake earlier when describing the default as I said it would default to mm-dd-yyyy when it actually defaults to mm/dd/yyyy so I fixed that as well.